mobility_data/importers/gas_filling_station.py

```diff
@@ -19,22 +23,35 @@
 class GasFillingStation:
 
     def __init__(self, elem, srid=settings.DEFAULT_SRID):
+        #Contains the complete address with zip_code and city
+        self.address = {}
+        #Contains Only steet_name and number
+        self.street_address = {}
         self.is_active = True
         attributes = elem.get("attributes")        
         x = attributes.get("LON",0)
         y = attributes.get("LAT",0)
         self.point = Point(x, y, srid=srid)              
         self.name = attributes.get("STATION_NAME", "")
-        self.address = attributes.get("ADDRESS", "")        
+        address_field = attributes.get("ADDRESS", "").lstrip().split(" ")        
+        street_name, street_number = get_street_name_and_number(address_field[0])
         self.zip_code = attributes.get("ZIP_CODE", "")
         self.city = attributes.get("CITY", "")      
-        self.operator = attributes.get("OPERATOR", "")
+        translated_street_names = get_street_name_translations(
+            street_name)               
+        for lang in LANGUAGES:
+            if street_number:
+                self.address[lang] = f"{translated_street_names[lang]} {street_number}, "
+                self.address[lang] += f"{self.zip_code} {self.city}"
+                self.street_address[lang] = f"{translated_street_names[lang]} {street_number}"
+            else:
+                self.address[lang] = f"{translated_street_names[lang]}, "
+                self.address[lang] += f"{self.zip_code} {self.city}"
```

Comment:
This entire statement could probably be refactored, especially line 49, can simply be popped out since it is constant throughout both conditions.